dog.py

```diff
@@ -501,8 +502,20 @@ def get_env_config() -> DogConfig:
         user_env = os.getenv('USER')
         if home_env:
             env_config[HOME] = home_env
+        else:
+            try:
+                env_config[HOME] = pwd.getpwuid(uid).pw_dir
+            except KeyError:
+                pass
```

Comment:
`KeyError` is raised by `getpwduid`, OK.
But the `pass` what will the consequences be for that?
Perhaps try to test if directory "/home/$USER" exists in the `except` clause and use that before `pass`ing. (I see this is done under the win32 case (does that work in windows?!))

Reply:
Thanks for the comments Mark.

I am a little hesitant in trying to do something other than just letting it run like normally and therefore fall back to "nobody" - in case someone is relying on that behavior.

I am also afraid of trying to assume too much (like home always being /home/$USER).

So I would rather leave it as is to avoid confusing existing users (even though they will of course be hit if the user the use are in "/etc/passwd" but without USER defined).

I found this bug on alpine which apparently does not set USER by default.